examples/hf_gptj6b.py

```diff
@@ -10,12 +10,13 @@
     'backend': {
         'service': 'gpt-j-6b',
         'remote': True,
-        'token': 'XXXXXXXXXXXXXXXXX' # Use your huggingface token
+        'token': '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' # Use your huggingface token
     },
     'newline-delimited': True, # Should a new line indicate the end of the completion
     'classes': ['positive', 'negative', 'neutral'], # defaults to None
     'max-length': 3, # output response upper length limit
-    'gpu': True, # True or the GPU number you would like to use (if local). False (or -1 if local) indicates CPU
+    'gpu': True, # True or the GPU number you would like to use (if local). False (or -1 if local) indicates CPU,
+    'resp_parser': 'text_gen_parser'
```

Comment:
you specify which parser should be used for the response, because responses from different models/tasks are different.

Reply:
Could these be documented some place? Docstrings, perhaps?

Reply:
Added these :+1: